src/common/wflign/src/wflign.cpp

```diff
@@ -352,7 +349,7 @@ void WFlign::wflign_affine_wavefront(
     // wflambda layer we then patch up the gaps between them
 
     float inception_score_max_ratio = 1.0 + 0.5 / mashmap_estimated_identity;
-    float max_mash_dist_to_evaluate = std::min(0.95, 0.05 / std::pow(mashmap_estimated_identity,5));
+    float max_mash_dist_to_evaluate = std::min(0.55, 0.05 / std::pow(mashmap_estimated_identity,13));
```

Comment:
What is this doing? Why 55%, why exponentiate to the 13th?

Reply:
With the previous formula it was not a problem, because with `mashmap_estimated_identity = 0.7` ( a very low estimated identity), we where obtaining `max_mash_dist_to_evaluate = 0,297495091`.

Now, by using 13th as exponential, we can have a bit more permissive `max_mash_dist_to_evaluate` values with high divergence sequence pairs (that is, low `mashmap_estimated_identity` values), without increasing them with low divergence pairs. But, very low estimated identities, like 0.7, would lead to values equal to 0.95, forcing the evaluation of lots of cells in the high-order DP-matrix. Using 0.55 as max would avoid that.